classes/Mail.php

```diff
@@ -298,7 +298,7 @@ function ($carry, $item) {
 
         if (is_array($to) && isset($to)) {
```

Comment:
```suggestion
        if (is_array($to) && isset($to)) {
            $to = array_map('strtolower', $to);
```

Reply:
The isset seems also useless :thinking:

Reply:
@PierreRambaud Correct

```suggestion
        if (is_array($to)) {
            $to = array_map('strtolower', $to);
```